src/index.ts

```diff
@@ -989,24 +1000,22 @@ export class FlashbotsBundleProvider extends providers.JsonRpcProvider {
     }
     const bundleTransactions = blockDetails.transactions
     const bundleCount = bundleTransactions[bundleTransactions.length - 1].bundle_index + 1
-    const signedPriorBundleTransactions = []
+    const signedPriorBundleTransactions: string[] = []
     for (let currentBundleId = 0; currentBundleId < bundleCount; currentBundleId++) {
       const currentBundleTransactions = bundleTransactions.filter((bundleTransaction) => bundleTransaction.bundle_index === currentBundleId)
       const currentBundleSignedTxs = await Promise.all(
         currentBundleTransactions.map(async (competitorBundleBlocksApiTx) => {
           const tx = await this.genericProvider.getTransaction(competitorBundleBlocksApiTx.transaction_hash)
-          if (tx.raw !== undefined) {
-            return tx.raw
-          }
-          if (tx.v !== undefined && tx.r !== undefined && tx.s !== undefined) {
+          // NOTE: The `raw` property no longer exists on TransactionResponse
+          // if (tx.raw !== undefined) {
+          //   return tx.raw
+          // }
```

Comment:
If the `raw` property no longer exists, should these comments be removed for simplicity ?